frontend/src/components/NumberInputWrapper.scss

```diff
@@ -0,0 +1,15 @@
+.odh-number-input-wrapper {
+  &.m-full-width {
```

Comment:
Why not push back to UX if the the PF component doesn't support full width?
is there a bug logged to PF and does PF approve of such designs?

Reply:
I can log a request for PF support. I think it looks better and is more usable at full width so I didn't want to push back. Also, I am seeing an issue with the units value wrapping for some reason, hence the:
```
    .pf-v5-c-number-input__unit {
      white-space: nowrap;
    }
```